proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -90,6 +90,7 @@ message WorkflowExecutionInfo {
     int64 last_event_task_id = 17;
     int64 last_first_event_id = 18;
     int64 last_completed_workflow_task_started_event_id = 19;
+    // Deprecated. use `WorkflowExecutionState.start_time`
```

Comment:
plz create a task for tracking the deprecation work.

Reply:
done